error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/JUnitToAssertJRules.java

```diff
@@ -302,6 +272,436 @@
 final class JUnitToAssertJRules {
   private JUnitToAssertJRules() {}
 
+  static final class AssertThatBooleanArrayContainsExactly {
+    @BeforeTemplate
+    void before(boolean[] actual, boolean[] expected) {
+      assertArrayEquals(expected, actual);
+    }
+
+    @AfterTemplate
+    @UseImportPolicy(STATIC_IMPORT_ALWAYS)
+    void after(boolean[] actual, boolean[] expected) {
+      assertThat(actual).containsExactly(expected);
+    }
+  }
+
+  static final class AssertThatBooleanArrayWithFailMessageContainsExactly {
+    @BeforeTemplate
+    void before(boolean[] actual, String message, boolean[] expected) {
+      assertArrayEquals(expected, actual, message);
+    }
+
+    @AfterTemplate
+    @UseImportPolicy(STATIC_IMPORT_ALWAYS)
+    void after(boolean[] actual, String message, boolean[] expected) {
+      assertThat(actual).withFailMessage(message).containsExactly(expected);
```

Comment:
I wanted to suggest this to make it consistent with `AssertJThrowingCallableRules` where we do the message assertion after the checking the expected value.
```suggestion
      assertThat(actual).containsExactly(expected).withFailMessage(message);
```
But Intellij warns `Result of 'AbstractAssert. withFailMessage()' is ignored `; which I think is because of `@CheckReturnValue` annotation.

Reply:
Yep, see also the Javadoc of `AbstractAssert#withFailMessage`:
```
* You must set the message <b>before</b> calling the assertion otherwise it is ignored as the failing assertion breaks
* the call chain by throwing an {@link AssertionError}.
```

Reply:
That makes more sense 😄